packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1278,22 +1268,44 @@ export function createLodestarMetrics(
         name: "lodestar_shuffling_cache_size",
         help: "Shuffling cache size",
       }),
-      processStateInsertNew: register.gauge({
-        name: "lodestar_shuffling_cache_process_state_insert_new_total",
-        help: "Total number of times processState is called resulting a new shuffling",
-      }),
-      processStateUpdatePromise: register.gauge({
-        name: "lodestar_shuffling_cache_process_state_update_promise_total",
-        help: "Total number of times processState is called resulting a promise being updated with shuffling",
-      }),
-      processStateNoOp: register.gauge({
-        name: "lodestar_shuffling_cache_process_state_no_op_total",
-        help: "Total number of times processState is called resulting no changes",
-      }),
       insertPromiseCount: register.gauge({
         name: "lodestar_shuffling_cache_insert_promise_count",
         help: "Total number of times insertPromise is called",
       }),
+      hit: register.gauge({
+        name: "lodestar_gossip_attestation_shuffling_cache_hit_count",
+        help: "Count of shuffling cache hit",
+      }),
+      miss: register.gauge({
+        name: "lodestar_gossip_attestation_shuffling_cache_miss_count",
+        help: "Count of shuffling cache miss",
+      }),
+      shufflingRecalculated: register.gauge({
+        name: "lodestar_shuffling_cache_recalculated_shuffling_count",
+        help: "Count of shuffling cache promises that were discarded and the shuffling was built synchronously",
+      }),
+      shufflingPromiseNotResolvedAndThrownAway: register.gauge({
+        name: "lodestar_shuffling_cache_promise_not_resolved_and_thrown_away_count",
+        help: "Count of shuffling cache promises that were discarded and the shuffling was built synchronously",
+      }),
+      shufflingPromiseNotResolved: register.gauge({
+        name: "lodestar_shuffling_cache_promise_not_resolved_count",
+        help: "Count of shuffling cache promises that were requested before the promise was resolved",
+      }),
+      nextShufflingNotOnEpochCache: register.gauge({
+        name: "lodestar_shuffling_cache_next_shuffling_not_on_epoch_cache",
+        help: "The next shuffling was not on the epoch cache before the epoch transition",
+      }),
+      shufflingPromiseResolutionTime: register.histogram({
+        name: "lodestar_shuffling_cache_promise_resolution_time",
+        help: "Count of shuffling cache promises that were requested before the promise was resolved",
```

Comment:
`help` does not seem match the metric name

Reply:
also need to tweak `buckets` to more realistic values to benefit us. If this gets called only in the private `set()` function in the constructor, it may not worth to track

Reply:
Good catch.  I updated a couple to make them more descriptive